package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.1.0",
   "license": "MIT",
   "scripts": {
-    "lint": "eslint . --ext .ts --fix",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
```

Comment:
So we need to change this in every repo 🙈 and currently everyone assumes running lint command will fix

Then maybe can we leave `lint` command like before but add a new command `lint:error` or something like that? 😄 just a thought but if not we should change everywhere

Reply:
We could have a `lint:nofix` instead, and keep all the existing config happy.